python/feathub/processors/flink/table_builder/source_sink_utils.py

```diff
@@ -80,28 +84,36 @@ def get_table_from_source(
         raise FeathubException(f"Unsupported source type {type(source)}.")
 
 
-def insert_into_sink(
+def add_sink_to_statement_set(
```

Comment:
In use cases when users only need to materialize one feature descriptor at one time, would it bring performance overhead by using statement sets?